sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -300,14 +297,14 @@ public void clearTransaction() {
    * @return the transaction
    */
   @Nullable
-  public ITransaction getTransaction() {
-    return this.transaction;
+  public ISpan getSpan() {
```

Comment:
so never about about https://github.com/getsentry/sentry-java/pull/1286/files#r583536154

the thing is, what previously was returning the latest active Span is returning the root span, same method but different behavior, I guess this would cause a lot of problems, people might fix the compilation error due to return type, but it'll mess up their span hierarchy.